Gemfile

```diff
@@ -2,7 +2,7 @@
 
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
-ruby "3.1.4"
+ruby "3.1.4" unless ENV["K8"] == "yes"
```

Comment:
@dkinzer I'm confused by this part.  We had to update the Ruby version the other day because the docker image was using 3.1.4.  What is it using now?